drupal-org.make

```diff
@@ -10,14 +10,14 @@ projects[entity][subdir] = "contrib"
 projects[entityreference][version] = 1.1
 projects[entityreference][subdir] = "contrib"
 projects[entityreference][patch][] = "http://drupal.org/files/1580348-universal-formatters-17.patch"
-projects[rules][version] = 2.7
+projects[rules][version] = 2.8
```

Comment:
Ok. I will look at rules. However, tests passes so can you tell where we
see it breaking so a test can be created.
Den 26/01/2015 16.09 skrev "Bojan Zivanovic" <notifications@github.com>:

> In drupal-org.make
> <https://github.com/commerceguys/commerce_kickstart/pull/84#discussion_r23535988>
> :
>
> > @@ -10,14 +10,14 @@ projects[entity][subdir] = "contrib"
> >  projects[entityreference][version] = 1.1
> >  projects[entityreference][subdir] = "contrib"
> >  projects[entityreference][patch][] = "http://drupal.org/files/1580348-universal-formatters-17.patch"
> > -projects[rules][version] = 2.7
> > +projects[rules][version] = 2.8
>
> No. Rules 2.8 doesn't work with Commerce.
> https://www.drupal.org/node/2403851
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/commerceguys/commerce_kickstart/pull/84/files#r23535988>
> .
>